src/interactive/documentation.ts

```diff
@@ -0,0 +1,228 @@
+import * as path from 'path'
+import * as vscode from 'vscode'
+import { withLanguageClient } from '../extension'
+import { getParamsAtPosition, setContext } from '../utils'
+
+const viewType = 'JuliaDocumentationBrowser'
+const panelActiveContextKey = 'juliaDocumentationPaneActive'
+let extensionPath: string = undefined
```

Comment:
```suggestion
let extensionPath: string | undefined = undefined
```
Just in preparation for strict mode :)